### PR TITLE
Attempting to fix backspace issue in new UI

### PR DESCRIPTION
**Pull request by jherico**
This solution isn't perfect, because the backspace shortcut won't work at all if any of the new UI dialogs are on the screen, and I'm still getting debug output related to the backspace shortcut when I use it in the QML UI.  However, I believe these problems will go away with my next PR which is refactoring the menu and shortcuts system to move all menu mechanisms into QML, so I think it's a reasonable approach in the short term

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: ae804357d884835946b37ee1a980c54ff929618c available at: https://gist.github.com/6797ac7188b6de0b5052<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4682.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4682.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: ebc5ec2200daa5f817d9f71c922498ba3d97dccb available at: https://gist.github.com/6797ac7188b6de0b5052<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4682.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4682.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: a4619c8e59c08dfc21df3ba2fa45fb054437a801 available at: https://gist.github.com/6797ac7188b6de0b5052<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4682.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4682.dmg

File: libraries/render-utils/src/OffscreenUi.cpp [hunk 1]
```diff
@@ -280,12 +312,14 @@ bool OffscreenUi::eventFilter(QObject* originalDestination, QEvent* event) {
...
             }
-            return false;
+            break;
         }
 
         case QEvent::KeyPress:
         case QEvent::KeyRelease: {
             event->ignore();
+            //if (_quickWindow->activeFocusItem()) {
+            //    _quickWindow->sendEvent(_quickWindow->activeFocusItem(), event);
```
Comment (Atlante45) [inline libraries/render-utils/src/OffscreenUi.cpp]: Do you want to keep that in?

Comment (jherico) [inline libraries/render-utils/src/OffscreenUi.cpp]: no.  removed.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0a9398808f2e140f77e4d631fecf78c517e9aa42 available at: https://gist.github.com/6797ac7188b6de0b5052<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4682.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4682.dmg

**Pull request merged**